travis/format.sh

```diff
@@ -61,10 +61,10 @@ if [[ $FAILED_CHECKS -ne 0 ]]; then
   exit 1
 fi
 
-FILETYPES="*.c *.cc *.cpp *.h *.m *.mm *.dart"
+FILETYPES="*.dart"
 
 set +e
-TRAILING_SPACES=$(git diff $DIFF_OPTS $BASE_SHA..HEAD -- $FILETYPES | xargs grep --line-number --with-filename '\s\+$')
+TRAILING_SPACES=$(git diff $DIFF_OPTS $BASE_SHA..HEAD -- $FILETYPES | xargs grep --line-number --with-filename '[[:blank:]]\+$')
```

Comment:
I think you also need `--extended-regexp` as an option for the character classes to work (but maybe it works without that).